irrigation_datparser/parser.py

```diff
@@ -0,0 +1,48 @@
+import datetime
+import dateutil.tz
+import csv
+import json
+
+
+def gallon2mm(value):
+    if value:
+	#      gallons -> lit = kg -> kg m-2 s-1
+        return (int(value)*3.78541)/((20*200)*(24*60*60))
+    return 0.0
+
+
+GEOMETRY = {
```

Comment:
shouldn't this geomtery come from the sensor geometry, so that they match and only have to be updated in one place?